File: app/src/main/java/za/co/woolworths/financial/services/android/checkout/service/network/ProductCountMap.java

package za.co.woolworths.financial.services.android.checkout.service.network;

import com.google.gson.annotations.SerializedName;

public class ProductCountMap {

    @SerializedName("totalProductCount")
    private Integer totalProductCount;

    public Integer getTotalProductCount() {
        return totalProductCount;
    }

    public void setTotalProductCount(Integer totalProductCount) {
        this.totalProductCount = totalProductCount;
    }

}

File: app/src/main/java/za/co/woolworths/financial/services/android/geolocation/model/ProductCountMap.kt
package za.co.woolworths.financial.services.android.geolocation.model

data class ProductCountMap(
    val foodProductCartItemIdMap: FoodProductCartItemIdMap,
    val productItemCountMap: ProductItemCountMap,
    val quantityLimit: QuantityLimit,
    val totalProductCount: Int
)